nerdlets/shared/utils/query-formatter.js

```diff
@@ -65,7 +89,7 @@ const getFilterValue = (value, isString, escapeType) => {
 }
 
 const escapeValue = value => {
-  return value.replace(/'/g, "\\\'")
+  return value.replace(/'/g, "\\'")
```

Comment:
@aso1124 I can't remember changing this line. Is this something we changed while on a call?

Reply:
Yeah, maybe by accident - this should be \\\